dotcom-rendering/src/layouts/ImmersiveLayout.tsx

```diff
@@ -639,39 +639,48 @@ export const ImmersiveLayout = (props: WebProps | AppProps) => {
 										article.imagesForAppsLightbox
 									}
 								/>
-								{showBodyEndSlot && (
-									<Island
-										priority="feature"
-										defer={{ until: 'visible' }}
-										clientOnly={true}
-									>
-										<SlotBodyEnd
-											contentType={article.contentType}
-											contributionsServiceUrl={
-												contributionsServiceUrl
-											}
-											idApiUrl={article.config.idApiUrl}
-											isMinuteArticle={
-												article.pageType.isMinuteArticle
-											}
-											isPaidContent={
-												article.pageType.isPaidContent
-											}
-											keywordIds={
-												article.config.keywordIds
-											}
-											pageId={article.pageId}
-											sectionId={article.config.section}
-											shouldHideReaderRevenue={
-												article.shouldHideReaderRevenue
-											}
-											stage={article.config.stage}
-											tags={article.tags}
-											renderAds={renderAds}
-											isLabs={isLabs}
-										/>
-									</Island>
-								)}
+								{renderingTarget === 'Web' &&
+									showBodyEndSlot && (
```

Comment:
Can we inline this check as part of the `showBodyEndSlot` definition?

Reply:
Nice! This actually dropped the number of added lines significantly 👌